modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheAdapter.java

```diff
@@ -4641,20 +4641,20 @@ private void advance() {
         // TODO https://ggsystems.atlassian.net/browse/GG-38173
         if (!isDhtAtomic()) {
             throw new UnsupportedOperationException(
-                "Operation is not supported on " +
-                    (isNear() ? "near " : "") + (isLocal() ? "local " : "") + ctx.config().getAtomicityMode() +
-                    " cache [name=" + name() + ']');
+                    "Operation is not supported on " +
```

Comment:
Additional indentation is not needed. Please take a look at https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-BracesandIndentation